foundationsync/indico_foundationsync.py

```diff
@@ -104,9 +112,9 @@ def _parse_room_data(self, raw_data, coordinates, room_id):
         data['capacity'] = int(raw_data['CAPACITY']) if raw_data['CAPACITY'] else None
         data['surface_area'] = int(raw_data['SURFACE']) if raw_data['SURFACE'] else None
         data['division'] = raw_data.get('DEPARTMENT')
-        data['telephone'] = raw_data.get('TELEPHONE')
-        data['key_location'] = raw_data.get('WHERE_IS_KEY')
-        data['comments'] = raw_data.get('COMMENTS')
+        data['telephone'] = raw_data.get('TELEPHONE') or ''
+        data['key_location'] = self._html_to_markdown(raw_data.get('WHERE_IS_KEY')) if raw_data.get('WHERE_IS_KEY') else ''
+        data['comments'] = self._html_to_markdown(raw_data.get('COMMENTS')) if raw_data.get('COMMENTS') else ''
         data['is_reservable'] = raw_data['IS_RESERVABLE'] != 'N'
```

Comment:
This one changed: Rooms that are not marked as reservable in Indico are now also not reservable by its managers. So this column shouldn't be updated anymore (it should already be True)